docs/profiles.md

```diff
@@ -336,27 +337,80 @@ profile `my_dep` using the name `my_res2`.
 
 # Profile Attributes
 
-Attributes may be used in profiles to define secrets, such as user names and passwords, that should not otherwise be stored in plain-text in a cookbook. First specify a variable in the control for each secret, then add the secret to a YAML file located on the local machine, and then run `inspec exec` and specify the path to that Yaml file using the `--attrs` attribute.
+Attributes may be used in profiles to define secrets, such as user names and passwords, that should not otherwise be stored in plain-text in a cookbook. Attributes may be set for the whole profile in the `inspec.yml`.
```

Comment:
I know you're just building on what is already there, but this is too limiting of a description what attributes are for - they provide no facilities for secrets, but are frequently used to parameterize a profile for use in different environments / targets.